assets/src/components/v2/pre_fare_single_screen_alert.tsx

```diff
@@ -40,11 +45,19 @@ const standardLayout = (
   remedy: string,
   effect: string,
   location: string | null,
+  bannerHeight: number,
   disruptionDiagram?: DisruptionDiagramData
 ) => {
+  const maxTextHeight =
+    SCREEN_HEIGHT -
+    FOOTER_HEIGHT -
+    BOTTOM_MARGIN -
+    ALERT_CARD_PADDING -
+    bannerHeight;
```

Comment:
For some reason I feel like I would understand this better as `SCREEN_HEIGHT - (FOOTER_HEIGHT + BOTTOM_MARGIN + ALERT_CARD_PADDING + bannerHeight)`, but that might be just me

Reply:
I agree, that does make more sense. Bundled that change in with the merge.